scripts/data_prep/convert_delta_to_json.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2022 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import logging
+import os
+
+import pandas as pd
+from databricks import sql
+
+log = logging.getLogger(__name__)
+
+
+def stream_delta_to_json(args: argparse.Namespace, batch_size: int = 1 << 20):
```

Comment:
Should `batch_size` be exposed via the script?

Reply:
It should be automatically adjusted based on available memory and dataset. I'm still thinking how to set this value more properly. It should probably not be exposed.